finalproject-AVC/backend/.env.example

```
@@ -0,0 +1,16 @@
+NODE_ENV=development
+
+# Server
+PORT=3000
+HOST=0.0.0.0
+
+# Database (Docker PostgreSQL - use docker-compose up -d to start)
+DATABASE_URL="postgresql://usuario:password@localhost:5432/sc_padel_db?schema=public"
+
+# JWT
+JWT_SECRET=your-super-secret-jwt-key-change-this-in-production
+JWT_EXPIRES_IN=7d
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_EXPIRES_IN key should go before the JWT_SECRET key

(UnorderedKey)